packages/lego-bricks/package.json

```diff
@@ -72,7 +72,7 @@
     "@storybook/react": "^8.6.4",
     "@storybook/react-vite": "^8.6.4",
     "@storybook/test": "^8.6.4",
-    "@types/express": "^5.0.0",
+    "@types/express": "^4.17.21",
```

Comment:
Why does this need to be a lower version?

Reply:
It was wrong, we use express v4. Doesn't seem like it mattered much, but best to use correct version of types:)